frontend/public/data/possibleFilters.js

```diff
@@ -4,28 +4,38 @@ import GroupIcon from "@material-ui/icons/Group";
 import ExploreOutlinedIcon from "@material-ui/icons/ExploreOutlined";
 import GroupAddIcon from "@material-ui/icons/GroupAdd";
 import countries from "./countries.json";
+
 export default function getFilters(key, filterChoices) {
-  if (key === "projects") return getProjectsFilters(filterChoices);
-  else if (key === "organizations") return getOrganizationsFilters(filterChoices);
-  else if (key === "members") return getMembersFilters(filterChoices);
-  else console.log("possibleFilters invalid input:" + key);
+  if (!filterChoices) {
+    throw new Error("No filter choices supplied");
+  }
```

Comment:
Guarded against not having `filterChoices` and broke out the conditionals below.